src/app/App.tsx

```diff
@@ -45,6 +45,7 @@ BigNumber.config({
   FORMAT: {
     decimalSeparator,
     groupSeparator: groupingSeparator,
+    groupSize: 3,
```

Comment:
I wonder if there's a weird locale that doesn't use 3.

Reply:
I guess there are locales that have other group sizes, e.g., for India, it's 3 for 1000s, then 2 for everything else. Although this would be a big refactor to support this, as react-native-localize doesn't support this.